editor/components/inserter/group.js

```diff
@@ -48,7 +48,7 @@ export default class InserterGroup extends Component {
 				role="menuitem"
 				key={ block.name }
 				className="editor-inserter__block"
-				onClick={ selectBlock( block.name ) }
+				onClick={ selectBlock( block.name, block.attributes ) }
```

Comment:
Maybe we should just pass the `block` object as an argument here instead of two arguments.

Reply:
👌 fixed in 8ced4510795d9e428a2f6d1c2bbb4e2f744da23b.